impress-org/givewp
# Feature: Add Pro tip message to Payment Gateway Settings

## Description

This PR includes a dismissible "Pro tip" recommendation for the Fee Recovery addon that is placed after the "Test Mode" section in the Payment Gateway tab on the Settings page. The recommendation should be visible unless the user has the add-on active and/or has previously dismissed it as an option.

PR has been updated to include text suggestions from Loom comments for fee recovery promotions*

## Affects
Settings 
Payment Gateway

## Visuals



## Testing Instructions

- Navigate to the Settings page
- While on the Payment Gateway tab view the recommendation message
- Activate Fee Recovery addon
- Verify the recommendation is no longer displayed
- Deactivate the add on and revisit the page
- Test the UTM link
- Dismiss the Option
- Refresh the page verify the recommendation remains closed

## Pre-review Checklist
